src/tests/ftest/rebuild/container_create.yaml

```diff
@@ -0,0 +1,43 @@
+hosts:
+  test_machines:
+    - boro-A
+    - boro-B
+    - boro-C
+    - boro-D
+    - boro-E
+    - boro-F
+timeout: 180
+testparams:
+  kill_rank: !mux
+    rank_zero:
+      rank: 0
+    rank_one:
+      rank: 1
+    rank_two:
+      rank: 2
+    rank_three:
+      rank: 3
+    rank_four:
+      rank: 4
+    rank_five:
+      rank: 5
+server_config:
+  server_name: daos_server
```

Comment:
This name needs to match with control plane conf key so please change server_name >> name,

Reply:
Done.